packages/loader/container-loader/src/container.ts

```diff
@@ -1335,7 +1335,14 @@ export class Container
 						async (summary) => {
 							// Actually go and create the resolved document
 							if (this.service === undefined) {
-								const createNewResolvedUrl = await this.urlResolver.resolve(request);
+								const newRequest: IRequest = {
```

Comment:
does this need to be in container? Can it instead be in the createCreateNewRequestion function of the odsp driver? as this is odsp specific.

Reply:
loaded code details in publicly available on the container:
https://github.com/microsoft/FluidFramework/blob/b70e394731dba9d6f324f744e59818bd56746dc5/packages/loader/container-loader/src/container.ts#L717

The call of attach would be in charge of adding it if they want it, rather than it being done here

Reply:
The IFluidCodeDetails exists in the container-definitions and driver does not depends on that. So, it will be little harder. What we can do is to expose the packageName which is of type IContainerPackageInfo on the container and the apps can supply that directly to the create request create method. 
ContainerPackageInfo = getPackageName(IFluidCodeDetails),
we do expose getpackageName method but apps would need to do an extra step of calling it before supplying it into the create request method. So, to avoid that we can expose iContainerPackageInfo on the container class just like we do IFluidCodeDetails.